app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -34,6 +34,7 @@ def proxy_command_options(template_invocation, host)
             :name => host.name,
             :check_mode => host.host_param('ansible_roles_check_mode'),
             :job_check_mode => template_invocation.template.ansible_check_mode,
+            :job_diff_mode => template_invocation.template.ansible_diff_mode,
```

Comment:
Just looking at the pattern here: don't we need to add `diff_mode` here as well?..